tests/admin_inlines/admin.py

```diff
@@ -257,3 +271,7 @@ class SomeChildModelInline(admin.TabularInline):
 site.register(ExtraTerrestrial, inlines=[SightingInline])
 site.register(SomeParentModel, inlines=[SomeChildModelInline])
 site.register([Question, Inner4Stacked, Inner4Tabular])
+
+# For #29961
```

Comment:
Ticket reference not necessary. Actually, I'm not sure this is the right place for the tests considering it doesn't really have anything to do with admin inlines. I would limit the test to admin_widgets as long as you can also test the template changes there.

Reply:
Moving the tests to admin_widgets is indeed a better idea. Template changes can be tested more easily and specifically there.